.github/workflows/publish.yml

```
@@ -126,10 +126,15 @@ jobs:
         timeout-minutes: 2
 
       - name: Upload Snap
-        run: snapcraft upload $(cat ${{ runner.temp }}/snap-releases-args) snap/Parsec_${{ steps.version.outputs.full }}_linux_*.snap
+        run: |
+          set -x
+          for channel_arg in $(cat ${{ runner.temp }}/snap-releases-args); do
+            snapcraft upload $channel_arg snap/Parsec_${{ steps.version.outputs.full }}_linux_*.snap
```

Now that `snapcraft upload` is called 1 time for each channel, I think is clearer if we extract the `--release=` part _out_ of the `misc/snapcraft_releases.py` script. 
The script will therefore print only the channels to be updated, and this line will be: 

```suggestion
          for channel_arg in $(cat ${{ runner.temp }}/snap-releases-args); do
            snapcraft upload --release=$channel_arg snap/Parsec_${{ steps.version.outputs.full }}_linux_*.snap
```

---

done